docs/en/engines/table-engines/special/alias.md

```diff
@@ -0,0 +1,53 @@
+---
+description: 'Create an alias of a table.'
+sidebar_label: 'Alias'
+sidebar_position: 120
+slug: /en/engines/table-engines/special/alias
+title: 'Alias Table Engine'
+---
+
+# Alias Table Engine
+
+The Alias table engine is a reference to another table.
+
+## Usage in ClickHouse Server {#usage-in-clickhouse-server}
+
+```sql
+ENGINE = Alias(database_name, table_name)
```

Comment:
it would be nice if we also support `Alias(database_name.table_name)`. It's more user friendly

Reply:
> it would be nice if we also support `Alias(database_name.table_name)`. It's more user friendly

Fine.